patches/tModLoader/Terraria/Projectile.cs.patch

```diff
@@ -1285,6 +1295,41 @@
  			if (num3 < 24f)
  				Kill();
  
+@@ -36463,7 +_,8 @@
+ 				Main.tile[point4.X, point4.Y] = new Tile();
+ 
+ 			bool flag = true;
+-			if (Main.tile[point4.X, point4.Y].nactive() && AI_007_GrapplingHooks_CanTileBeLatchedOnTo(Main.tile[point4.X, point4.Y]))
++			// TML: Moved nactive check into method, changed from tile to coords
++			if (AI_007_GrapplingHooks_CanTileBeLatchedOnTo(point4.X, point4.Y))
+ 				flag = false;
```

Comment:
(See next comment) Here, you can do `flag = ProjectileLoader.GrappleCanLatchOnTo(this, point4.X, point4.Y) ?? flag`